lms/envs/common.py

```diff
@@ -340,9 +340,6 @@
     # Enable OpenBadge support. See the BADGR_* settings later in this file.
     'ENABLE_OPENBADGES': False,
 
-    # The block types to disable need to be specified in "x block disable config" in django admin.
-    'ENABLE_DISABLING_XBLOCK_TYPES': True,
```

Comment:
@cpennington I am ensuring that the new configuration models are populated with the correct data before the code is deployed. So yes, a data migration is happening, though through the script that will be used to update the models in the future.

Currently there are only a couple of entries specified so the database could also be manually updated, but I wanted to make sure the update process is smooth before starting to use the code.